src/Controller/UI/ReportResearchGroupDatasetStatusController.php

```diff
@@ -323,4 +327,35 @@ private function getVersionTwoReport(Collection $datasets, array $options): arra
             'dataArray' => $dataArray
         );
     }
+
+    /**
+     * Get custom dataset status string for the version two report.
+     *
+     * @param Dataset $dataset An instance of dataset entity.
+     *
+     * @return string
+     */
+    private function getDatasetStatus(Dataset $dataset): string
```

Comment:
This is a necessary evil, the getStatus from the Dataset entity if there only to support another report, and not used for anything else.

One day we shall refactor all the status things.